vendor/k8s.io/kubernetes/test/integration/namespace/ns_conditions_test.go

```diff
@@ -98,10 +97,10 @@ func TestNamespaceCondition(t *testing.T) {
 			if condition.Type == corev1.NamespaceDeletionContentFailure && condition.Message == `All content successfully deleted, may be waiting on finalization` {
 				conditionsFound++
 			}
-			if condition.Type == corev1.NamespaceContentRemaining && condition.Message == `Some resources are remaining: deployments.apps has 1 resource instances` {
+			if condition.Type == corev1.NamespaceContentRemaining && condition.Message == `Some resources are remaining: deployments.apps has 1 resource instances, deployments.extensions has 1 resource instances` {
```

Comment:
this is a side effect of having multiple versions of the same resource, which can be misleading.